tad/api/routes/projects.py

```diff
@@ -0,0 +1,56 @@
+import logging
+from typing import Annotated
+
+from fastapi import APIRouter, Depends, Request
+from fastapi.responses import HTMLResponse
+
+from tad.api.deps import templates
+from tad.core.config import get_settings
+from tad.core.exceptions import UnsafeFileError
+from tad.schema.instrument import InstrumentMetadataBase
+from tad.schema.project import ProjectNew
+from tad.schema.system_card import SystemCard
+from tad.services.instruments import InstrumentsService
+from tad.services.projects import ProjectsService
+from tad.services.storage import StorageFactory
+from tad.utils.path import is_safe_path
+
+router = APIRouter()
+logger = logging.getLogger(__name__)
+
+
+@router.get("/new")
+async def get_new(
+    request: Request,
+    instrument_service: Annotated[
+        InstrumentsService, Depends(InstrumentsService)
+    ],  # todo: create async_instruments_service
+) -> HTMLResponse:
+    instruments = await instrument_service.fetch_instruments()
+
+    return templates.TemplateResponse(request, "projects/new.html.j2", {"instruments": instruments})
+
+
+@router.post("/new", response_class=HTMLResponse)
+async def post_new(
+    request: Request,
+    project_new: ProjectNew,
+    projects_service: Annotated[ProjectsService, Depends(ProjectsService)],
+) -> HTMLResponse:
+    system_card_file = get_settings().CARD_DIR / f"{project_new.name}_system.yaml"
+    if not is_safe_path(get_settings().CARD_DIR, system_card_file):
+        raise UnsafeFileError(system_card_file)
+
+    project = projects_service.create(project_new.name, system_card_file)
+
+    instruments: list[InstrumentMetadataBase] = [
+        InstrumentMetadataBase(urn=instrument_urn) for instrument_urn in project_new.instruments
```

Comment:
When I have 1 instrument, this loops over the string instead of the list, and thus creates a system card containing instruments of 1 letter (being all the letters of the chosen instrument).